tools/preload.py

```diff
@@ -63,6 +63,8 @@ def get_absolute_url(domain, path, icon):
 
     if(path.startswith('http')):
         return icon_path
+    elif path_absolute_from_app_root:
```

Comment:
Nope it's really what I meant. I'm using the wording from the [MDN](https://developer.mozilla.org/en-US/Apps/Build/Manifest):
> Internal paths for manifests, icons, etc. must be absolute from the app's origin, not the root of the app. For example, if your manifest is at http://www.mysite.com/myapp/manifest.webapp, your install path will be /myapp/manifest.webapp, not /manifest.webapp.